.circleci/config.yml

```diff
@@ -647,6 +654,8 @@ workflows:
           parallel_num: 4
           env_name: "test"
           <<: *filter-master-branch
+          optional_steps:
```

Comment:
I don't follow why this is an "optional" step. Should this be blocking api-deploy-to-test, rather than puppeteer?